blog/blog/x/blog/genesis.go

```diff
@@ -4,19 +4,14 @@ import (
 	sdk "github.com/cosmos/cosmos-sdk/types"
 	"github.com/example/blog/x/blog/keeper"
 	"github.com/example/blog/x/blog/types"
-	// abci "github.com/tendermint/tendermint/abci/types"
 )
 
-// InitGenesis initialize default parameters
-// and the keeper's address to pubkey map
-func InitGenesis(ctx sdk.Context, k keeper.Keeper /* TODO: Define what keepers the module needs */, data types.GenesisState) {
-	// TODO: Define logic for when you would like to initalize a new genesis
+// InitGenesis initializes the capability module's state from a provided genesis
+// state.
+func InitGenesis(ctx sdk.Context, k keeper.Keeper, genState types.GenesisState) {
 }
 
-// ExportGenesis writes the current store values
-// to a genesis file, which can be imported again
-// with InitGenesis
-func ExportGenesis(ctx sdk.Context, k keeper.Keeper) (data types.GenesisState) {
-	// TODO: Define logic for exporting state
-	return types.NewGenesisState()
+// ExportGenesis returns the capability module's exported genesis.
```

Comment:
Is starport code gen taken from the capability module?

Reply:
That's a question for starport scaffold @ilgooz

Reply:
Not necessarily the cap module. We take sdk/simapp, sdk/x, gaia, akash and akash/x as a reference for different parts of the scaffold.

Reply:
Yep, definitelly! We need to review all the comments in source code and fix similar issues. Tracked by https://github.com/tendermint/starport/issues/665.

Reply:
Thanks, something for starport to look into